games-engines/openscenegraph/openscenegraph-3.4.1.recipe

```diff
@@ -0,0 +1,190 @@
+SUMMARY="High performance OpenGL-based 3D graphics toolkit"
+DESCRIPTION="The OpenSceneGraph is an open source high performance 3D \
+graphics toolkit, used by application developers in fields such as visual \
+simulation, games, virtual reality, scientific visualization and modelling. \
+Written entirely in Standard C++ and OpenGL it runs on all Windows platforms, \
+OSX, GNU/Linux, IRIX, Solaris, HP-Ux, AIX and FreeBSD operating systems. The \
+OpenSceneGraph is now well established as the world leading scene graph \
+technology, used widely in the vis-sim, space, scientific, oil-gas, games and \
+virtual reality industries."
+HOMEPAGE="http://www.openscenegraph.org/"
+COPYRIGHT="2002-2018 Robert Osfield and others"
```

Comment:
2002-2017

Reply:
It is still not resolved. This version released in 2017 according their release page, so it should be 2017.